src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Tables/ExtendedQueryTagDateTime.sql

```diff
@@ -25,7 +25,7 @@ CREATE TABLE dbo.ExtendedQueryTagDateTime (
 ) WITH (DATA_COMPRESSION = PAGE)
 
 -- Used in QIDO, PartitionKey is moved down to support cross partition query in future
-CREATE UNIQUE CLUSTERED INDEX IXC_ExtendedQueryTagDateTime ON dbo.ExtendedQueryTagDateTime
```

Comment:
Changing this would results in breaking changes for re-index operation. They rely on this being unique. Why would you remove this? We can throw exception when we have same tagvalues